lib/util/google-map-service.js

```diff
@@ -18,15 +18,17 @@ const processGeocode = data => {
 
 const processPlaces = data => {
     const { results } = data;
+
+    // keep names consistent
     return results.map(place => ({
-        placeId: place.place_id,
-        types: place.types,
+        place_id: place.place_id,
```

Comment:
Change this here to be consistent and make the `createLogs` function easier to deal with